app/src/main/java/org/oppia/android/app/testing/NavigationDrawerTestActivity.kt

```diff
@@ -37,17 +38,20 @@ class NavigationDrawerTestActivity :
   private var internalProfileId: Int = -1
 
   companion object {
-    fun createNavigationDrawerTestActivity(context: Context, profileId: Int?): Intent {
+    fun createNavigationDrawerTestActivity(context: Context, profileid: Int?): Intent {
       val intent = Intent(context, NavigationDrawerTestActivity::class.java)
-      intent.putExtra(NAVIGATION_PROFILE_ID_ARGUMENT_KEY, profileId)
+      val profileId = profileid?.let { ProfileId.newBuilder().setInternalId(it).build() }
+      if (profileId != null) {
+        intent.decorateWithUserProfileId(profileId)
+      }
```

Comment:
Differentiate between internalProfileId and profileId for clarity.

```suggestion
      val profileId = internalProfileId?.let { ProfileId.newBuilder().setInternalId(it).build() }
      if (profileId != null) {
        intent.decorateWithUserProfileId(profileId)
      }
```

Reply:
Okay.